app/views/step_by_step_pages/show.html.erb

```diff
@@ -90,15 +90,10 @@
     </div>
 
     <div class="app-side">
-      <%
-        metadata = {}
-        metadata.store("Status", @step_by_step_page.status[:text])
-        metadata.store("Last saved", format_full_date_and_time(@step_by_step_page.updated_at))
-        metadata["Last saved"] = "#{metadata["Last saved"]} by #{@step_by_step_page.assigned_to}" if @step_by_step_page.assigned_to.present?
-        metadata.store("Created", format_full_date_and_time(@step_by_step_page.created_at))
-        metadata.store("Links checked", format_full_date_and_time(@step_by_step_page.links_last_checked_date)) if @step_by_step_page.links_checked?
-      %>
-      <%= render "govuk_publishing_components/components/metadata", { other: metadata } %>
+      <%= render "govuk_publishing_components/components/summary_list", {
+        title: "Overview",
```

Comment:
The designs attached to the Trello card don't have "Overview" as a title. Have they changed?

Reply:
I wrongly assumed you have to provide a title, [according to the component docs](https://components.publishing.service.gov.uk/component-guide/select). Have just tried it without the title and it renders fine!

Will edit this PR and have raised a [PR to add a title-less example to the summary list component](https://github.com/alphagov/govuk_publishing_components/pull/1070).